pubspec.yaml

```diff
@@ -25,6 +26,7 @@ flutter:
 
   assets:
     - assets/images/
+    - assets/images/welcomeScreenImage.svg
```

Comment:
why are we adding the directory in multiple places?

Reply:
My bed. We can use either approach, such as **—assets/images/** **—assets/images/welcomeScreenImage.svg**. 
The first approach is better if there are multiple files